### ISSUE TITLE
Accessibility: Make drag-and-drop accessible via keyboard

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#181732]
CC @meganrogge @isidorn

Recent VSCode updates have introduced some drag-and-drop mouse interactions to automatically insert image markdown syntax, etc.

I was wondering if VSCode team had any keyboard alternatives for these interactions. If not, please consider providing keyboard workarounds.

**Comment by meganrogge** [comment]
@mjbvz I recall raising this issue, but cannot remember what you said in response. Do keyboard users have a way to do something like this?

There is a command: `Markdown: Insert Image from Workspace` but not sure that does the same thing since it requires you to navigate to the exact file path

**Comment by mjbvz** [comment]
Copy paste is the keyboard only version of drag and drop. We're still actively building out its functionality but it should work very similar to drop going forward
